### PR TITLE
Docs m1

### PR DESCRIPTION
**Pull request by tmbo**
**Proposed changes**:
- revert changes done for docker support. currently not possible
- added documentation for M1 limitations
- added changelog

Fixes https://rasahq.atlassian.net/browse/ATO-325

### CODE REVIEW

File: docs/docs/installation.mdx [hunk 1]
```diff
@@ -316,3 +323,25 @@ To download a specific version, specify the version number:
...
+to the follow restrictions
+
+* Duckling can currently not be run as a docker container on Apple Silicon. If you want
+  to use the [duckling entity extractor](components.mdx#ducklingentityextractor) we recommend a cloud deployment of 
+  duckling. Progress on this can be tracked on the 
+  [Duckling project](https://github.com/facebook/duckling/issues/695).
+* Rasa can not be run in a Docker container on Apple Silicon, only native installations
+  currently work. An installation in docker requires support for Ubuntu aarch64
+  which the current tensorflow version 2.8 does not provide - only MacOS is 
+  supported as an opperating system running on aarch64. We expect a future
```
Comment (tmbo) [inline docs/docs/installation.mdx:342]:
```suggestion
  supported as an operating system running on aarch64. We expect a future
```

**Comment by tmbo** [comment]
@znat do I need to do anything special to merge this ontop of your changes?

**Review by rasa-jmac — APPROVED** [review]

File: docs/docs/components.mdx [hunk 2]
```diff
@@ -495,6 +495,8 @@ Note: The `feature-dimension` for sequence and sentence features does not have t
   :::note
   To use `ConveRTFeaturizer`, install Rasa Open Source with `pip3 install rasa[convert]`.
 
+  The optional dependencies required for this component can currently not be 
+  installed on MacOS running on M1 / M2.
```
Comment (ancalita) [inline docs/docs/components.mdx:499]:
```suggestion
  Note that this component cannot currently run on MacOS using M1 / M2 architecture. 
  More information on this limitation is available [here](./installation.mdx#m1--m2-apple-silicon-limitations).
```

File: docs/docs/installation.mdx [hunk 3]
```diff
@@ -38,6 +38,17 @@ your machine and continue development locally.
...
   <TabItem value="macos">
 
+:::info
+
+Rasa supports **x86_64 (Intel) and M1 / M2 (Apple Silicon)** architectures.
+The support is for development purposes only and we strongly recommend 
+production deployments to use Ubuntu / Linux on x86_64.
+
+Installations on Apple Silicon have some 
+[limitations listed further down](#m1--m2-apple-silicon-limitations).
```
Comment (ancalita) [inline docs/docs/installation.mdx:48]:
```suggestion
More information about the limitations of rasa installations on Apple Silicon can be found
in this [section](#m1--m2-apple-silicon-limitations).
```

File: docs/docs/installation.mdx [hunk 4]
```diff
@@ -316,3 +323,25 @@ To download a specific version, specify the version number:
 ```bash
 pip3 install rasa==3.0
 ```
+
+## M1 / M2 (Apple Silicon) Limitations
+
+Rasa installations on Apple Silicon use Apple Metal leading to reduced training times
```
Comment (ancalita) [inline docs/docs/installation.mdx:329]: What is Apple Metal? This might need a clarification.

File: docs/docs/installation.mdx [hunk 5]
```diff
@@ -316,3 +323,25 @@ To download a specific version, specify the version number:
...
+for local development.
+
+Currently, not all of Rasa's dependencies support Apple Silicon natively. This leads
+to the follow restrictions
```
Comment (ancalita) [inline docs/docs/installation.mdx:333]:
```suggestion
to the follow restrictions:
```

File: docs/docs/installation.mdx [hunk 6]
```diff
@@ -316,3 +323,25 @@ To download a specific version, specify the version number:
...
+
+* Duckling can currently not be run as a docker container on Apple Silicon. If you want
```
Comment (ancalita) [inline docs/docs/installation.mdx:335]:
```suggestion
* You can not run Duckling as a docker container on Apple Silicon. If you want
```

File: docs/docs/installation.mdx [hunk 7]
```diff
@@ -316,3 +323,25 @@ To download a specific version, specify the version number:
...
+  to use the [duckling entity extractor](components.mdx#ducklingentityextractor) we recommend a cloud deployment of 
+  duckling. Progress on this can be tracked on the 
+  [Duckling project](https://github.com/facebook/duckling/issues/695).
+* Rasa can not be run in a Docker container on Apple Silicon, only native installations
```
Comment (ancalita) [inline docs/docs/installation.mdx:339]:
```suggestion
* You cannot run Rasa in a Docker container on Apple Silicon, only native installations
```

**Review by ancalita — COMMENTED** [review]
Left a few rephrasing suggestions 👍🏻

File: docs/docs/installation.mdx (hunk 4 above)
Comment (tmbo) [reply docs/docs/installation.mdx:329]: its Apples ML / GPU acceleration, I'll add a link :+1:

```suggestion
Rasa installations on Apple Silicon use [Apple Metal](https://developer.apple.com/metal/) leading to reduced training times
```

File: docs/docs/components.mdx (hunk 2 above)
Comment (tmbo) [reply docs/docs/components.mdx]: great suggestion 🙏

**Review by ancalita — APPROVED** [review]

File: docs/docs/installation.mdx [hunk 8]
```diff
@@ -316,3 +323,25 @@ To download a specific version, specify the version number:
 ```bash
 pip3 install rasa==3.0
 ```
+
+## M1 / M2 (Apple Silicon) Limitations
+
+Rasa installations on Apple Silicon use [Apple Metal](https://developer.apple.com/metal/) leading to reduced training times
```
Comment (twerkmeister) [inline docs/docs/installation.mdx:329]: We should make metal optional, and change this

File: docs/docs/installation.mdx [hunk 9]
```diff
@@ -316,3 +323,25 @@ To download a specific version, specify the version number:
...
+* You can not run Duckling as a docker container on Apple Silicon. If you want
+  to use the [duckling entity extractor](components.mdx#ducklingentityextractor) we recommend a cloud deployment of 
+  duckling. Progress on this can be tracked on the 
+  [Duckling project](https://github.com/facebook/duckling/issues/695).
+* You cannot run Rasa in a Docker container on Apple Silicon, only native installations
+  currently work. An installation in docker requires support for Ubuntu aarch64
+  which the current tensorflow version 2.8 does not provide - only MacOS is 
+  supported as an operating system running on aarch64. We expect a future
+  upgrade of Tensorflow to allow Apple Silicon users to run Rasa inside of Docker.
+* Rasa on Apple Silicon does not support the [`ConveRTFeaturizer` component](components.mdx#convertfeaturizer) or pipelines
```
Comment (twerkmeister) [inline docs/docs/installation.mdx:344]: Also spacy isn't supported until we upgrade to 3.4.1.

File: docs/docs/installation.mdx (hunk 8 above)
Comment (tmbo) [reply docs/docs/installation.mdx:329]: do you mind doing the change directly on this branch?

**Comment by tmbo** [comment]
I am going to wait until we've addressed the spacy issues before merging this.

**Pull request merged**